app/controllers/cases/case_transitions_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Cases
+  class CaseTransitionsController < CasesController
+    def create
+      @case = Case::Base.find(params[:case_id])
+      authorize @case
+
+      @case.state_machine.mark_as_waiting_for_data!(acting_user: current_user, acting_team: current_user.managing_teams.first)
+    end
```

Comment:
@mseedat-moj Line 7 gives `NoMethodError: undefined method valid? for nil:NilClass` when you run `spec/controllers/cases/case_transitions_controller_spec.rb` - spent ages digging around trying to see what nil is getting valid? sent to it. Checked out the before filters in the model with no luck. Can you see why this is happening? I might be sending in the wrong things to `mark_as_waiting_for_data!` but it seemed about right to me.